### PR TITLE
Feat / Delete Tracks 🆘

### PR DESCRIPTION
**Pull request by johnxgaehner**
I know it's hard, but sometimes you have to go separate ways - also with your tracks.
Therefore you can now remove a track from your collection. 😢

Also clicked sometimes accidentally on Reset instead of Submit.
Might be because every Submit/Ok/Yes button is on the right side....

### CODE REVIEW

File: src/pages/AddToPlaylistPage.js [hunk 1]
```diff
@@ -56,11 +59,29 @@ export default function AddToPlaylistPage() {
     setPlaylists(updatedPlaylists);
   }
 
+  function onDeleteTrackClick() {
+    const confirmBox = window.confirm(
+      "Do you really want to delete this track?"
+    );
+    if (confirmBox === true) {
```
Comment (mikemangas) [inline src/pages/AddToPlaylistPage.js:66]: i guess, you can just delete the === true

File: src/pages/AddToPlaylistPage.js [hunk 2]
```diff
@@ -56,11 +59,29 @@ export default function AddToPlaylistPage() {
...
+        return track.id !== id;
+      });
+      setTracks(patchedCollection);
+      history.goBack();
+    }
+  }
+
   return (
     <section className="AddToPlaylistPage">
+      <div onClick={onDeleteTrackClick} className="ATPP__DeleteTrack">
```
Comment (mikemangas) [inline src/pages/AddToPlaylistPage.js:78]: is there a reason, why delete track starts with capital D?

File: src/pages/AddToPlaylistPage.js [hunk 3]
```diff
@@ -56,11 +59,29 @@ export default function AddToPlaylistPage() {
...
+        <p>DELETE TRACK</p>
+      </div>
       <Link to="/create-new-playlist" className="Row--flat --accented">
         <p>CREATE NEW PLAYLIST</p>
```
Comment (mikemangas) [inline src/pages/AddToPlaylistPage.js:82]: would give it some a more semantic tag here, like h3 or smthng like that?

File: src/pages/AnalysePage.js [hunk 4]
```diff
@@ -103,10 +103,10 @@ export default function AnalysePage() {
           />
         </div>
         <div className="Row--flat --space-between">
```
Comment (mikemangas) [inline src/pages/AnalysePage.js:105]: nice naming here, i guess :)

**Comment by mikemangas** [comment]
general feedback on the user experience:

your apps looks really nice and simple - which i really like. but sometimes  i have difficulties, to find something that i need, like going to the menu, submitting or reseting something etc. maybe some visual feedback would help?! 
Example:
in your collection site, the arrow is a bit hard to identify... but it is a call to action place (button)... therefore a bit more visual feedback would be great... like color, size, highlighting etc..

File: src/pages/AddToPlaylistPage.js (hunk 1 above)
Comment (johnxgaehner) [reply src/pages/AddToPlaylistPage.js:66]: true!

File: src/pages/AddToPlaylistPage.js (hunk 2 above)
Comment (johnxgaehner) [reply src/pages/AddToPlaylistPage.js:78]: started my own naming convention. haha

**Pull request merged**